src/cdi_ndi/NDI_CP_Eloss.cc

```diff
@@ -0,0 +1,288 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   cdi_ndi/NDI_CP_Eloss.cc
+ * \author Ben R. Ryan
+ * \date   2020 Jun 3
+ * \brief  NDI_CP_Eloss member definitions.
+ * \note   Copyright (C) 2019-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#include "NDI_CP_Eloss.hh"
+#include "ds++/DracoStrings.hh"
+#include "ds++/Soft_Equivalence.hh"
+
+#include <cmath>
+
+namespace rtt_cdi_ndi {
+
+//------------------------------------------------------------------------------------------------//
+/*!
+ * \brief Do a 3D linear interpolation between vertices of a rectangular prism.
+ *
+ * Algorithm from wikipedia's Trilinear Interpolation article, hat tip to E.
+ * Norris for the reference.
+ *
+ * \param[in] x0   lower x coordinate of lattice
+ * \param[in] x1   upper x coordinate of lattic
```

Comment:
Lattic -> Lattice here, and on line 31 :)

Reply:
Cosmic rays flipping bits in my RAM leading to random spelling errors no doubt